lib/generators/hyrax/work/templates/feature_spec.rb.erb

```diff
@@ -3,7 +3,8 @@
 require 'rails_helper'
 include Warden::Test::Helpers
 
-RSpec.feature 'Create a <%= class_name %>' do
+# NOTE: If you generated more than one work, you have to set "js: true"
```

Comment:
This feature spec is generated by Hyrax's work generator. This is telling you that if you have run the work generator more than once, set `js` to `true`.